inspirehep/demosite/data/collection.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
What about creating a collection directory or a `fixtures` one, in the root. A bit like we currently have `kbs` and `taxonomies`. The thing is that in principle we could remove the `demosite` directory and everything should run smoothly. But this particular file is really needed in order to implement the production configuration.